authors/apps/authentication/serializers.py

```diff
@@ -141,3 +251,41 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ResetSerializer(serializers.Serializer):
+    email = serializers.EmailField()
+    token = serializers.CharField(max_length=255, read_only=True)
+
+    # List the fields that should b
+
+    def validate(self, data):
+        """
+        Validate the email enterd by the user that requests to reset account passaword
```

Comment:
I noticed the word "entered" is spelt "enterd" on this line. Please correct that. Please consider installing a spell check extension on your code editor to catch such things in future. If using VSCode, you can try https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
@evansmusomi, thank you for noticing that.